### PR TITLE
Fabrizio test

### PR DESCRIPTION
**Pull request by fabriziobertocci**
Ok I gave up with the rebase... 'git rebase' kept not showing the merge commits. Any attempt to push would be rejected and any 'git pull' would cause duplication of the commits.
I have created a new branch (fabrizio-test) and apparently this branch seems 'clean enough'.
Let's see if this works.

### CODE REVIEW

File: libpicotcp/CMakeLists.txt [hunk 1]
```diff
@@ -67,3 +72,21 @@ if(LibPicotcp)
     set_property(TARGET picotcp PROPERTY INTERFACE_LINK_LIBRARIES picotcp_sources)
     target_include_directories(picotcp INTERFACE "${SOURCE_DIR}/build/include")
 endif()
+if(LibPicotcpBsd)
```
Comment (latentPrion) [inline libpicotcp/CMakeLists.txt]: Hey, minor request: could you nest this if block (`LibPicotcpBsd`) within the previous block (`LibPicotcp`)?

Comment (latentPrion) [reply libpicotcp/CMakeLists.txt]: Ah, please ignore this comment - I see what happened here, this isn't your change

Comment (latentPrion) [reply libpicotcp/CMakeLists.txt]: I'll just pull in the branch on my account and merge it -- please consider this PR to be closed; sorry for the turbulence but I'm mostly new to dealing with PRs on Github (I've used Bitbucket so far) so I'm missing details in the interface.

Comment (latentPrion) [reply libpicotcp/CMakeLists.txt]: https://github.com/seL4/util_libs/pull/18 <-- If you're fine with the final state here, @fabriziobertocci , I'm fine with merging~

**Comment by lsf37** [comment]
Closing this because it was merged in #18

**Pull request closed without merging**